bc_obps/service/application_access_service.py

```diff
@@ -46,22 +47,22 @@ def is_user_eligible_to_request_admin_access(
 
         Args:
             user_guid (uuid): The user for whom eligibility is being checked.
-            operatorIid (uuid): The if of the operator to which admin access is being requested.
+            operator_id (uuid): The if of the operator to which admin access is being requested.
 
         Returns:
             True or raises an exception.
         """
         approved_admins = UserOperatorDataAccessService.get_admin_users(operator_id, UserOperator.Statuses.APPROVED)
         if approved_admins.filter(user_guid=user_guid).exists():
-            raise Exception("You are already an admin for this Operator!")
+            raise UserError("You are already an admin for this Operator!")
         if len(approved_admins) > 0:
-            raise Exception("This Operator already has an admin user!")
+            raise UserError("This Operator already has an admin user!")
         # User already has a pending request for this operator
         # NOTE: This is a bit of a weird case, but it's possible for a user to have a pending request for an operator and if we show the UserOperator request form, they could submit another request and end up with two
         pending_admins = UserOperatorDataAccessService.get_admin_users(operator_id, UserOperator.Statuses.PENDING)
 
         if pending_admins.filter(user_guid=user_guid).exists():
-            raise Exception("You already have a pending request for this Operator!")
+            raise UserError("You already have a pending request for this Operator!")
```

Comment:
It's fun how some of the errors have exclamation marks. You didn't just do this wrong. You did this wrong!

Reply:
I think I did most of them 🙃